modules/programs/alot-accounts.nix

```diff
@@ -25,7 +25,7 @@ with lib;
   config = mkIf config.notmuch.enable {
     alot.sendMailCommand = mkOptionDefault (
       if config.msmtp.enable
-      then cfg.msmtp.sendCommand
+      then "msmtpq --read-envelope-from --read-recipients"
```

Comment:
Why don't we factor the msmtp.sendCommand ? It can still be overriden.

Reply:
I was thinking of instead adding a more general option `accounts.email.accounts.<name>.sendMailCommand`, which would be defined something like "a command prefix for a sendmail compatible".

The msmtp module could, if enabled, populate this option at `mkDefault` level but if some other smtp module is added then this could also set the option. Or the user could set the value themself, of course.

But I would prefer to do this separately from this PR.